### PR TITLE
update 0.10.3

### PR DESCRIPTION
**Pull request by avasomething**
Changelog:
- bump version, checksum
- cleaned up and fixed broken tests
- updated dependencies
- added `build/skip`

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,38 +1,46 @@
...
+    - pip
   imports:
     - pathspec
   commands:
-    - pytest --pyargs pathspec  # [unix]
+    - pip check
+    - cd tests && pytest --verbose  # [unix]
     # skip symlink tests on windows
     # https://github.com/cpburnz/python-path-specification/commit/d268375a4c6aa
-    - pytest --pyargs pathspec -k "not test_2_1_check_realpath and not test_2_4_recursive_links and not test_2_5_recursive_circular_links"  # [win]
```
Comment (katietz) [inline recipe/meta.yaml:34]: I would add here a ' || cmd /K "exit /b 0"' to this line, so check on Windows is not causing a build failure.  And move comments up.

**Review by katietz — CHANGES_REQUESTED** [review]
beside this, fine.

File: recipe/meta.yaml (hunk 1 above)
Comment (danpetry) [reply recipe/meta.yaml:34]: The test passes on Windows btw

**Review by danpetry — APPROVED** [review]
Nice work, thanks!

**Review by ELundby45 — APPROVED** [review]
LGTM!

**Pull request merged**